File: proyectSaludVital_Angular/src/app/DTO/Expediente-medico-dto.ts
import { EntradaHistorialDTO } from "./paciente-ver-expedi";


export interface ExpedienteMedicoDTO {
  id: number;
  nombrePaciente: string;
  numeroIdentificacion: string;
  fechaNacimiento: string;
  nombreMedico: string;
  entradasHistorial: EntradaHistorialDTO[];
}

File: ProyectoSaludVital-Backend/src/main/java/com/example/practica/DTO/ExpedienteMedicoDTO.java
package com.example.practica.DTO;

import java.time.LocalDate;
import java.util.List;

import lombok.*;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class ExpedienteMedicoDTO {

    private Long id;
    private String nombrePaciente;
    private String numeroIdentificacion;
    private LocalDate fechaNacimiento;
    private String nombreMedico;
    private List<EntradaHistorialDTO> entradasHistorial;
}
